pkg/maps/configmap/configmap.go

```diff
@@ -161,6 +163,32 @@ func (m *EndpointConfigMap) Update(value *EndpointConfig) error {
 	return m.Map.Update(configKey, value)
 }
 
+// Close closes the FD of the given CofigMap. Returns an error if the close
```

Comment:
This shouldn't be necessary; the generic Map close here should take care of this in the right way:

https://github.com/cilium/cilium/blob/0c1dbad17c2ddb13697d889787b8f0bac3f2fc0f/pkg/bpf/map_linux.go#L362-L378

Maybe it's the fact that the Map is not embedded directly into `EndpointConfigMap`? Ie, these lines:

https://github.com/cilium/cilium/blob/0c1dbad17c2ddb13697d889787b8f0bac3f2fc0f/pkg/maps/configmap/configmap.go#L137-L141

Should have the 

```
	Map  *bpf.Map
```

become 
```
	*bpf.Map
```